egs2/librispeech_100/asr1_biasing/decode.sh

```diff
@@ -0,0 +1,40 @@
+. ./path.sh
+. ./cmd.sh
+export PYTHONPATH="/home/gs534/rds/rds-t2-cs164-KQ4S3rlDzm8/gs534/opensource/espnet:$PYTHONPATH"
```

Comment:
This path contains your local information. As I suggested above, you can merge this into `asr.sh`.

Reply:
Will remove this decode.sh file in a later push.